src/routes/categories/components/CategoryListItem.css

```diff
@@ -43,3 +43,7 @@
     border-bottom: 1px solid var(--theme-color);
   }
 }
+
+.highlight {
+  background-color: var(--theme-color);
```

Comment:
Should we use here a lighter color? or maybe no color at all and just bolden it?

Reply:
@Schedulaar

Reply:
Not sure, I think just bold font would be enough...

Reply:
Just bold font:
![image](https://user-images.githubusercontent.com/29095487/37038290-b31b8db2-2154-11e8-87f4-f61c141cd2d8.png)
little bit unobstrusive imo

Reply:
I think it's obtrusive ( :D ) enough because the user knows what he has typed and is only interested in looking through the search results. With more highlighting the user concentrates more on the search key (which he already knows) and less on the content I think. 🤔

Reply:
sounds logic ^^